packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -882,6 +918,94 @@ void main() {
     final Material itemMaterial = tester.widget<Material>(buttonMaterial);
     expect(itemMaterial.color, themeData.colorScheme.onSurface.withOpacity(0.12));
   });
+
+  testWidgets('The default text input field should not be focused on mobile platforms '
+      'when it is tapped', (WidgetTester tester) async {
+    final ThemeData themeData = ThemeData();
+
+    Widget buildDropdownMenu({bool? requestFocusOnTap}) => MaterialApp(
+      theme: themeData,
+      home: Scaffold(
+        body: Column(
+          children: <Widget>[
+            DropdownMenu<TestMenu>(
+              requestFocusOnTap: requestFocusOnTap,
+              dropdownMenuEntries: menuChildren,
+            ),
+          ],
+        ),
+      ),
+    );
+
+    // Test default condition.
+    await tester.pumpWidget(buildDropdownMenu());
+    await tester.pump();
+
+    final Finder textFieldFinder = find.byType(TextField);
+    final TextField result = tester.firstWidget<TextField>(textFieldFinder);
```

Comment:
We should only use `firstWidget` when more than one widget is expected and the first one is the one we want. Otherwise `tester.widget`